src/frontend/apps/standalone_site/src/features/ClassRooms/components/Create/ClassRoomCreateForm.tsx

```diff
@@ -0,0 +1,189 @@
+import {
+  Form,
+  FormField,
+  Text,
+  TextArea,
+  TextInput,
+  Button,
+  Select,
+  Box,
+} from 'grommet';
+import { Alert } from 'grommet-icons';
+import { useCreateClassroom } from 'lib-classroom';
+import { Fragment, useState } from 'react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+
+import { usePlaylists } from 'features/Playlist';
+
+const messages = defineMessages({
+  titleLabel: {
+    defaultMessage: 'Title',
+    description: 'Label for title in classroom creation form.',
+    id: 'component.ClassroomCreateForm.titleLabel',
+  },
+  descriptionLabel: {
+    defaultMessage: 'Description',
+    description: 'Label for description in classroom creation form.',
+    id: 'component.ClassroomCreateForm.descriptionLabel',
+  },
+  welcomeTextLabel: {
+    defaultMessage: 'Welcome text',
+    description: 'Label for welcome text in classroom creation form.',
+    id: 'component.ClassroomCreateForm.welcomeTextLabel',
+  },
+  requiredField: {
+    defaultMessage: 'This field is required to create the classroom.',
+    description: 'Message when classroom field is missing.',
+    id: 'component.ClassroomCreateForm.requiredField',
+  },
+  selectPlaylistLabel: {
+    defaultMessage: 'Choose the playlist.',
+    description: 'Label select playlist.',
+    id: 'component.ClassroomCreateForm.selectPlaylistLabel',
+  },
+  submitLabel: {
+    defaultMessage: 'Add classroom',
+    description: 'Label for button submit in classroom creation form.',
+    id: 'component.ClassroomCreateForm.submitLabel',
+  },
+  Error: {
+    defaultMessage: 'Sorry, an error has occurred. Please try again.',
+    description: 'Text when there is an error.',
+    id: 'features.ClassroomCreateForm.Error',
+  },
+});
+
+type ClassroomCreate = {
+  playlist: string;
+  title: string;
+  description?: string;
+};
+
+interface ClassroomCreateFormProps {
+  onSubmit?: () => void;
+}
+
+const ClassroomCreateForm = ({ onSubmit }: ClassroomCreateFormProps) => {
+  const intl = useIntl();
+  const { data, error: errorPlaylist } = usePlaylists(
+    {
+      offset: '0',
+      limit: '0',
```

Comment:
You are not fetching any data here right ?

Reply:
These params are required, by doing that I get all the value, the limit pass to 999.
https://github.com/openfun/marsha/blob/master/src/frontend/packages/lib_components/src/common/queries/fetchList.tsx#L28